feconf.py

```diff
@@ -1056,6 +1056,8 @@ def get_empty_ratings():
 # Authentication method using GAE ID (google sign in)
 AUTH_METHOD_GAE = 'gae'
 
+# TODO(#10501): When can import domain objects in models layer move these back
```

Comment:
Once domain objects can be imported by the storage layer, move these ...

Reply:
Done.